components/buttons/ScrollButton.tsx

```diff
@@ -27,11 +27,12 @@ function ScrollButton() {
 
   return (
     <div className='fixed bottom-14 right-4 z-40 h-16 w-12'>
-      {backToTopButton && (
-        <button className='rounded-full bg-white shadow-md ' onClick={scrollUp}>
-          <img src={scrollImage} alt='scroll to top' />
-        </button>
-      )}
+      <button
+        className={`rounded-full bg-white shadow-md  ${backToTopButton ? 'visible' : 'pointer-events-none invisible'}`}
```

Comment:
Kind of confused as how these changes are leading to fix the error in safari?

Reply:
The issue was due to an empty true condition, which led to unexpected behavior, img copies in DOM, By updating this condition to execute a specific action when true, we effectively resolved the issue as highlighted by codeRabbit. The issue and solution videos illustrate this difference clearly.

Reply:
@akshatnema ??